src/main/java/net/minecraftforge/client/event/TextureStitchEvent.java

```diff
@@ -48,6 +48,16 @@ public TextureMap getMap()
     }
 
     /**
+     * Fired when the TextureMap is told to refresh it's stitched texture.
+     * Called after all textures for the map have been collected, but before the stitching
+     * is done.
+     */
+    public static class Collected extends TextureStitchEvent
+    {
+        public Collected(TextureMap map){ super(map); }
```

Comment:
Well, yes, all the events are improperly formatted. You can ignore this point, indeed.